packages/build/src/run-draft.ts

```diff
@@ -7,16 +7,23 @@ import { downloadArtifactFromEvergreen as downloadArtifactFromEvergreenFn } from
 import { generateChangelog as generateChangelogFn } from './git';
 import type { GithubRepo } from '@mongodb-js/devtools-github-repo';
 import { getPackageFile } from './packaging';
+import {
+  bumpAuxiliaryPackages as bumpAuxiliaryPackagesFn,
+  bumpMongoshReleasePackages as bumpMongoshReleasePackagesFn,
+} from './npm-packages/bump';
 
 export async function runDraft(
   config: Config,
   githubRepo: GithubRepo,
   uploadToDownloadCenter: typeof uploadArtifactToDownloadCenterFn = uploadArtifactToDownloadCenterFn,
   downloadArtifactFromEvergreen: typeof downloadArtifactFromEvergreenFn = downloadArtifactFromEvergreenFn,
-  ensureGithubReleaseExistsAndUpdateChangelog: typeof ensureGithubReleaseExistsAndUpdateChangelogFn = ensureGithubReleaseExistsAndUpdateChangelogFn
+  ensureGithubReleaseExistsAndUpdateChangelog: typeof ensureGithubReleaseExistsAndUpdateChangelogFn = ensureGithubReleaseExistsAndUpdateChangelogFn,
+  bumpMongoshReleasePackages: typeof bumpMongoshReleasePackagesFn = bumpMongoshReleasePackagesFn,
```

Comment:
I am definitely getting rid of this structure of functions the moment I have a chance but for now... 😫

Reply:
Yeah, please feel very free to create a ticket for this 🙂